README.md

```diff
@@ -0,0 +1,25 @@
+### Description
+
+Application for storing orders, allows to view, add, update and delete orders
+
+|                Path                   |    Method     |                   Description               |           Body example       |
+| --------------------------------------|---------------| --------------------------------------------| ---------------------------- |
+| `/orders/{id}`                        |    `DELETE`   |   delete order                              |                              |
+| `/orders`                             |    `POST`     |   create new order                          | {"Id":2,"Date":"2021-04-03T00:00:00Z","TableNumber":2,"WaiterId":2,"Price":226,"Payment":false}|
```

Comment:
could you show json body in the following way:
```json
Here goes your json object definition
```
I took this answer from here: https://stackoverflow.com/questions/14901245/how-to-style-a-json-block-in-github-wiki